src/please.go

```diff
@@ -171,7 +171,11 @@ var opts struct {
 	} `command:"cover" description:"Builds and tests one or more targets, and calculates coverage."`
 
 	Debug struct {
-		Port int               `short:"p" long:"port" description:"Debugging server port"`
+		Port  int `short:"p" long:"port" description:"Debugging server port"`
+		Share struct {
+			Network bool `long:"share_network" description:"Share network namespace"`
+			Mount   bool `long:"share_mount" description:"Share mount namespace"`
+		} `group:"Options to override mount and network namespacing on linux, if configured"`
```

Comment:
FYI that you could do
```
Mount   bool `long:"mount" description:"Share mount namespace"`
} `group:"...." namespace:"mount"`
```
doesn't matter much here but it's less repetitive when there are lots of them

Reply:
Fair! I copy-pasted the `exec` options in this case, but maybe could do some tidying.